types/resemblejs/index.d.ts

```diff
@@ -18,14 +18,20 @@ declare namespace Resemble {
   function outputSettings(settings: OutputSettings): typeof Resemble;
 
   interface OutputSettings {
-    errorColor: {
+    errorColor?: {
       red: number;
       green: number;
       blue: number;
     };
-    errorType: string;
-    transparency: number;
-    largeImageThreshold: number;
+    errorType?: string;
+    transparency?: number;
+    largeImageThreshold?: number;
+    ignoredBox?: {
+          left: number,
+          top: number,
+          right: number,
+          button: number
```

Comment:
Nitpicking: the indentation on these 4 doesn't match the rest of the file.

Reply:
`button`? I assume you mean `bottom`?